### ISSUE TITLE
Either Lens

### ISSUE THREAD

**Issue by okennedy** [UBOdin/mimir#169]
Create a new lens to support expressions of the form PICK_ONE(...)

Proposed syntax
```
CREATE LENS R_COLLAPSED
  AS SELECT PICK_ONE(A, B, C) FROM R 
  WITH PICKER
```

Optional thoughts...
* `PICK_ONE` : Allow a different choice for each row
* `PICK_ONE_GLOBALLY` : Effectively a form of Schema Matching
* `PICK_ONE_BY` : Maybe a group-by formulation of the same problem

Use case: 
* We have a dataset with columns A, B that conceptually represent the same thing.  In other words, you want to assume that A = B everywhere, but if that assumption is violated you want to be notified.

**Comment by mrb24** [comment]
Added in commit a0deb1ad314ba636956dfe1b78510471901f2b92 needs to be merged into master

**Comment by mrb24** [comment]
commit c39044ccbff153f38bf38b7c96266914a2729f5e closes #169
PickerLens merged into master
